### DISCUSSION TITLE
Unable to configure BOINC Manager on my Mac Studio M1 Max.

### DISCUSSION THREAD

**Discussion by Signalshifter** [BOINC/boinc#5242] [Q&A] [answered]
I have run BOINC Manager on my PC's for more than 14 years, without a hitch.  I now have a Mac Studio M1 Max. It is running just fine on this system but unlike the Windows version there are no configuration tabs at the top of the Simple View screen. I can find no place to go to configure how it runs, not even how to get to the advanced screen.  Right now it runs all of the time.  It does not impact my regular use but I would like to have it run just when the computer is idle.  

The MAC is running Ventura 13.3.1, BOINC Manger v7.20.4, wxWidgets version 3.1.5, I already had Rosetta 2 loaded  if that matters.

**Comment by AenBleidd** [comment]
Could you please try the new 7.22.1 version? If it doesn't work for you as well - please create a bug report here: https://github.com/BOINC/boinc/issues/new?assignees=&labels=&projects=&template=bug_report.md
And include this information together with some screenshots.

**Reply by Signalshifter** [reply]
Will do. and thanks

**Reply by AenBleidd** [reply, marked as answer]
Basically answered here: https://github.com/BOINC/boinc/issues/5243
